cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -306,3 +308,112 @@ def test_alert_run_ansible_playbook(full_template_vm_modscope, alert_profile, re
             timeout=60)
     except TimedOutError:
         pytest.fail("Ansible playbook method hasn't been executed.")
+
+
+@pytest.fixture(scope='module')
+def setup_ansible_repository(appliance, wait_for_ansible):
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name="billy",
+        url="https://github.com/billfitzgerald0120/ansible_playbooks",
+        description=fauxfactory.gen_alpha()
+    )
+    view = navigate_to(repository, "Details")
+    wait_for(
+        lambda: view.entities.summary("Properties").get_text_of("Status") == "successful",
+        timeout=60,
+        fail_func=view.toolbar.refresh.click
+    )
+    yield
+    repository.delete_if_exists()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1678132])
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize(
+    ("import_data", "item"),
+    ([DatastoreImport("automate_domain_bz_1678135.zip", "Ansible_State_Machine_for_Ansible_stats3",
+                      None, "using CFME creds"), "CatalogItemInitialization_jira23"],
+     [DatastoreImport("automate_domain_bz_1678135.zip", "Ansible_State_Machine_for_Ansible_stats3",
+                      None, "using CFME creds"), "CatalogItemInitialization_jira24"]),
+    ids=["method_to_playbook", "playbook_to_playbook"]
+)
+def test_variable_pass(request, appliance, setup_ansible_repository, import_datastore, import_data,
+                       item, dialog, catalog):
+    """
+    Bugzilla:
+        1678132
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        startsin: 5.11
+        setup:
+            1. Enable embedded ansible role
+        testSteps:
+            1. Add Ansible repo called billy -
+               https://github.com/ManageIQ/integration_tests_playbooks
+            2. Copy Export zip (Ansible_State_Machine_for_Ansible_stats3.zip ) to downloads
+               directory(Zip file with description - 'Automate domain' is attached with BZ(1678135)
+            3. Go to Automation/Automate Import/Export and import zip file
+            4. Click on "Toggle All/None" and hit the submit button
+            5. Go to Automation/Automate/Explorer and Enable the imported domain
+            6. Make sure all the playbook methods have all the information (see if Repository,
+               Playbook and Machine credentials have values), update if needed
```

Comment:
Looks like Step 1 to 6 are set up and only creating catalog item and ordering are test steps .